app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :Comments
+  has_many :likes_counter
+
+  after_save :updates_posts_counter
+
+  def updates_posts_counter
+    user.increment!(:PostsCounter)
+  end
```

Comment:
- [ ] [OPTIONAL] Because you are using the rails callback function `after_save` you can even make the `updates_posts_counter` private.

Reply:
thank you